vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/action/TouchActions.java

```diff
@@ -111,6 +113,34 @@ public void tap(WebElement element, Duration duration)
                 .perform();
     }
 
+    //CHECKSTYLE:OFF
+    /**
+     * Performs double tap on the <b>element</b>
+     * See details for
+     * <a href="https://github.com/appium/appium-xcuitest-driver#mobile-doubletap">iOS</a>,
+     * <a href="https://github.com/appium/appium-uiautomator2-driver/blob/master/docs/android-mobile-gestures.md#mobile-doubleclickgesture">Android</a>
+     * @param element element to tap, must not be {@code null}
+     */
+    //CHECKSTYLE:ON
+    public void doubleTap(WebElement element)
+    {
+        Optional<String> script = Optional.empty();
+        if (genericWebDriverManager.isAndroid())
+        {
+            script = Optional.of("mobile: doubleClickGesture");
+        }
+        else if (genericWebDriverManager.isIOS())
+        {
+            script = Optional.of("mobile: doubleTap");
+        }
+        script.ifPresent(s ->
+        {
+            Map<String, Object> args = Map.of("elementId",
+                    WebDriverUtils.unwrap(element, RemoteWebElement.class).getId());
+            webDriverProvider.getUnwrapped(JavascriptExecutor.class).executeScript(s, args);
+        });
```

Comment:
the action does nothing if the script is empty, should not we throw unsupported platform here?